.github/workflows/test-nix.yml

```diff
@@ -7,6 +7,11 @@ on:
   push:
     tags:
       - '*'
+  pull_request:
+    paths:
+      - 'flake.nix'
+      - 'flake.lock'
+      - 'package.nix'
```

Comment:
You might also want to run when there are changes to that workflow file (that tests nix build).

Reply:
Good point, done.